File: codes/BinaryTree/Python/BinaryTreeMain.py
import BinaryTreeModule as bm

bt1 = bm.BinaryTree()
bt2 = bm.BinaryTree()
bt3 = bm.BinaryTree()
bt4 = bm.BinaryTree()

bt1.SetData(1)
bt2.SetData(2)
bt3.SetData(3)
bt4.SetData(4)

bt1.MakeLeftSubTree(bt2)
bt1.MakeRightSubTree(bt3)
bt2.MakeLeftSubTree(bt4)

print(bt1.GetLeftSubTree().GetData())
print(bt1.GetLeftSubTree().GetLeftSubTree().GetData())

File: codes/BinaryTree/Java/BinaryTree/BinaryTreeMain.java
package BinaryTree;

public class BinaryTreeMain {
    public static void main(String[] args) {
        BinaryTree<Integer> bt1 = new BinaryTree<Integer>();
        BinaryTree<Integer> bt2 = new BinaryTree<Integer>();
        BinaryTree<Integer> bt3 = new BinaryTree<Integer>();
        BinaryTree<Integer> bt4 = new BinaryTree<Integer>();

        bt1.SetData(1);
        bt2.SetData(2);
        bt3.SetData(3);
        bt4.SetData(4);

        bt1.MakeLeftSubTree(bt2);
        bt1.MakeRightSubTree(bt3);
        bt2.MakeLeftSubTree(bt4);

        System.out.println(bt1.GetLeftSubTree().GetData());
        System.out.println(bt1.GetLeftSubTree().GetLeftSubTree().GetData());
    }
}
